index.css

```diff
@@ -0,0 +1,210 @@
+.homePage-container {
```

Comment:
Avoid mix naming convention for class names `homepage-container` or `home-page-container` would be good

Reply:
you mean to avoid camel case naming convention??

Reply:
> you mean to avoid camel case naming convention??

Yes, for classes don't use camel case